examples/api/test/cupertino/nav_bar/cupertino_sliver_nav_bar.0_test.dart

```diff
@@ -20,7 +20,7 @@ void main() {
     await tester.pumpAndSettle();
 
     // Large title is hidden and at higher position.
-    expect(tester.getBottomLeft(find.text('Contacts').first).dy, 36.0);
+    expect(tester.getBottomLeft(find.text('Contacts').first).dy, 36.0 + 8); // Static part + padding.
```

Comment:
nit: either move this to a variable as well or refer to the variable in the comment